app/frontend/src/services/commonFormService.js

```diff
@@ -1,7 +1,231 @@
+   * Fetch this form's settings
+   * @param {string} form The form name
+   * @returns {Promise} An axios response
+   */
+  getSettings(form) {
+    if (!isValidForm(form)) return Promise.reject('Invalid form specified');
+    return appAxios().get(`${form}/settings`);
+  },
+
+  /**
+   * @function getDashboardSettings
+   * Fetch a list of dashboard setting configurations
+   * @param {string} form The form name
+   * @returns {Promise} An axios response
+   */
+  getDashboardSettings(form) {
+    if (!isValidForm(form)) return Promise.reject('Invalid form specified');
+    return appAxios().get(`${form}/settings/dashboards`);
+  },
+
+  //
+  // Statuses
+  //
+
+  /**
+   * @function getStatusCodes
+   * Fetch the contents of the Status Codes lookup table
+   * @param {string} form The form name
+   * @returns {Promise} An axios response
+   */
+  getStatusCodes(form) {
+    if (!isValidForm(form)) return Promise.reject('Invalid form specified');
+    return appAxios().get(`${form}/current/statusCodes`);
+  },
+
+  /**
+   * @function getStatusCodes
+   * Fetch the contents of the Status Codes lookup table
+   * @param {string} form The form name
+   * @param {object} content An object containing the updated content for `form`
+   * @returns {Promise} An axios response
+   */
+  updateStatusCodes(form, content) {
+    if (!isValidForm(form)) return Promise.reject('Invalid form specified');
+    return appAxios().put(`${form}/current/statusCodes`, content);
+  },
+
+  /**
+   * @function getSubmissionStatuses
+   * Fetch the inspection statuses of a specific attestation form submission
+   * @param {string} form The form name
+   * @param {string} submissionId The guid of a submitted submissionId from the database
+   * @returns {Promise} An axios response
+   */
+  getSubmissionStatuses(form, submissionId) {
+    if (!isValidForm(form)) return Promise.reject('Invalid form specified');
+    return appAxios().get(`${form}/submissions/${submissionId}/statuses`);
+  },
+
+  /**
+   * @function sendSubmissionStatuses
+   * Update the statuses of a specific attestation form submission
+   * @param {string} form The form name
+   * @param {string} submissionId The guid of a submitted submission from the database
+   * @param {object} content An object representing the updated status for the `submissionId` form
+   * @returns {Promise} An axios response
+   */
+  sendSubmissionStatuses(form, submissionId, content) {
+    if (!isValidForm(form)) return Promise.reject('Invalid form specified');
+    return appAxios().post(`${form}/submissions/${submissionId}/statuses`, content);
+  },
+
+  //
+  // Submissions
+  //
+
+  /**
+   * @function getAllSubmissionData
+   * Fetch the contents of all attestation submission metadata
+   * @param {string} form The form name
+   * @param {boolean} [complete=false] Request complete metadata
+   * @returns {Promise} An axios response
+   */
+  getAllSubmissionData(form, complete = false) {
```

Comment:
Do you want to simplify this? 
just use tiny as the parameter and pass that into the BE?

Reply:
I just looked at this function again and inverted its default. It's now setting tiny=true by default, and only drops that parameter if and only if complete is set to true (aka I want all the data, not just metadata).

Reply:
Updated as discussed - passing in params by default instead for more caller flexibility.

Reply:
Updating the server side to have deleted=false and tiny=true by default on submission search.